manifests/l/LegacyInteractive/Launcher/1.12.0/LegacyInteractive.Launcher.locale.en-US.yaml

```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: LegacyInteractive.Launcher
+PackageVersion: 1.12.0
+PackageLocale: en-US
+Publisher: Legacy Interactive, Inc.
+PackageName: Legacy Games Launcher
+License: Copyright (c) 2023 Legacy Games

```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate